src/instantiator/characteristic/enumeration-characteristic-instantiator.ts

```diff
@@ -124,4 +130,32 @@ export class EnumerationCharacteristicInstantiator extends CharacteristicInstant
     shouldProcess(nameNode: NamedNode): boolean {
         return this.metaModelElementInstantiator.sammC.EnumerationCharacteristic().equals(nameNode);
     }
+
+    solveBlankNodeValues(resolvedBlankNodes) {
+        const resolvedBlankNodesValues = [];
+        if (resolvedBlankNodes.length > 0) {
+            resolvedBlankNodes.forEach(item => {
+                resolvedBlankNodesValues.push({ value: item.object.value, language: item.object.language });
+            });
+        }
+        if (resolvedBlankNodesValues.length) {
+            return this.retrieveStringFromArrayOfValues(resolvedBlankNodesValues);
+        }
+        else {
+            return '';
+        }
+    }
+    retrieveStringFromArrayOfValues(values) {
```

Comment:
Please remove this, it makes no sense here and only returns a string.

It would be better to return what was there before as an array with an object so that it can be filtered better.

Reply:
done